lib/hotkeys.ts

```diff
@@ -0,0 +1,24 @@
+const NON_ENGLISH_LAYOUT = /^[^\x00-\x7F]$/
+
+export function convertKey(e: KeyboardEvent): string {
+  if (NON_ENGLISH_LAYOUT.test(e.key) && /^Key.$/.test(e.code)) {
+    return e.code.replace(/^Key/, '').toLowerCase()
+  }
+  return e.key.toLowerCase()
+}
+
+function compareHotkey(hotkey: string, e: KeyboardEvent): boolean {
+  let prefix = ''
```

Comment:
I small note. You are going this job again and again for every hokey in `isHotkey` function

So, I speed it up and cleaned the code 71f3b5e